src/main/java/Command.java

```diff
@@ -0,0 +1,24 @@
+import java.util.ArrayList;
+
+public class Command {
+    public CommandType type;
+    public ArrayList<String> args;
```

Comment:
Is it possible to make this protected? The coding standard says class variables should not be public